docs/cookbook.md

```diff
@@ -0,0 +1,378 @@
+# Cookbook
+
+(basic-table-operations)=
+
+## Basic table operations
+
+### Read in a file to a specific format
```

Comment:
The original intent was that you could just do:
`tables_io.read(<some_file>)` and have it work.   I think it would be good to add that as the first operation here.